docs/howto/import.md

```diff
@@ -16,47 +16,27 @@ redirect_from:
 
-### Importing using the lakeFS UI
+### Using the lakeFS UI
 
-#### Prerequisites
-
-lakeFS must have permissions to list the objects at the source object store,
-and in the same region of your destination bucket.
-
-lakeFS supports two ways to ingest objects from the object store without copying the data:
-
-1. [Importing using the lakeFS UI](#importing-using-the-lakefs-ui) - A UI dialog to trigger an import to a designated import branch. It creates a commit from all imported objects.
-1. [Importing using lakectl cli](#importing-using-lakectl-cli) - You can use the [`lakectl` CLI command](../reference/cli.md#lakectl) to create uncommitted objects in a branch. It will make sequential calls between the CLI and the server.
-
-#### Using the import wizard
-
-Clicking the Import button from any branch will open the following dialog:
-
-![Import dialog example configured with S3](../assets/img/UI-Import-Dialog.png)
-
-If it's the first import to the selected branch, it will create the import branch named `_<branch_name>_imported`.
-lakeFS will import all objects from the Source URI to the import branch under the given prefix.
-
-The UI will update periodically with the amount of objects imported. How long it takes depends on the amount of objects to be imported but will roughly be a few thousand objects per second.
-
-![img.png](../assets/img/ui-import-waiting.png)
-
-Once the import is completed, you can merge the changes from the import branch to the source branch.
-
-![img.png](../assets/img/ui-import-completed.png)
-
-
-### Importing using lakectl cli
+To import using the UI, lakeFS must have permissions to list the objects in the source object store.
+{: .note }
 
-The `lakectl` cli supports _import_ and _ingest_ commands to import objects from an external source.
+1. In your repository's main page, click the _Import_ button to open the import dialog:
+   ![img.png](../assets/img/UI-Import-Dialog.png)
+2. Under _Import from_, fill in the location on your object store you would like to import from.
+3. Fill in the import destination in lakeFS and a commit message.
+4. Once the import is completed, you can merge the changes from the import branch to the source branch.
 
-- The _import_ command acts the same as the UI import wizard. It imports (zero copy) and commits the changes on `_<branch_name>_imported` branch with an optional flag to also merge the changes to `<branch_name>`.
-- The _Ingest_ is listing the source bucket (and optional prefix) from the client, and creating pointers to the returned objects in lakeFS. The objects will be staged on the branch.
+#### Notes
+{: .no_toc }
 
+* On the first import to a branch, a dedicated branch named `_<branch_name>_imported` will be created. lakeFS will import all objects to this branch under the given prefix.
+* The import duration depends on the amount of imported objects, but will roughly be a few thousand objects per second.
 
-#### Using the `lakectl import` command
+### _lakectl import_
```

Comment:
nit: If H3 is "Using the lakeFS UI" then the next H3 should be "Using lakectl CLI" and then the two options here H4. Either that, or at least indicate in the two `lakectl` H3s that it's a CLI option.

Reply:
While this is logically correct, I prefer to avoid the cognitive load of another nested level.